docs/index.md

```diff
@@ -1,49 +1,29 @@
-<img src="images/logo.svg" alt="logo" width="200"/>
+<img src="icon-probabilistic.png" alt="logo" width="200"/>
 
 # RedisBloom - Probablistic Datatypes Module for Redis
 
-This module provides three datatypes, a Scalable **Bloom Filter**, **Cuckoo Filter** and a **Count-Mins-Sketch**.
-The first two datatypes are used to determine (with a given degree of certainty) whether
-an item is present or absent from a collection while the last is used to count the
-frequency of the different items in sub-linear space.
-
+RedisBloom module provides four datatypes, a Scalable **Bloom Filter** and **Cuckoo Filter**, a **Count-Mins-Sketch** and a **Top-K**.
+**Bloom and Cuckoo filters** are used to determine (with a given degree of certainty) whether an item is present or absent from a collection. While **Count-Min Sketch** is used to count the frequency of the elephant items in sub-linear space and **Top-K** maintains a list of K most frequent items.
```

Comment:
I believe the explanation of count-min sketch focuses on the implementation where the others focus on explain what they enable.